lib/miq_tools_services/github/message_builder.rb

```diff
@@ -0,0 +1,56 @@
+require 'stringio'
+
+module MiqToolsServices
+  class Github
+    class MessageBuilder
```

Comment:
I see MessageBuilder and I think it generates a Message when it really generates a chain of messages.  Maybe MessageChainBuilder?